### ISSUE TITLE
Does not draw with Android 4.1

### ISSUE THREAD

**Issue by shnrch** [pents90/svg-android#7]
Hi everyone!

I am using the library successfully in my apps and I really like it. But unfortunately it seems that something is different with Android 4.1 - the canvas elements parsed from svg files does not come up, while it is working fine with a different device.

Any suggestions on this?
Since I haven't seen any complains by other people it still ma be a problem related with my svg files or the rest of the code I am using to display..

Steffen

**Comment by pents90** [comment]
Hi, this is because of a bug with the Android hardware acceleration. See here for the solution: http://stackoverflow.com/questions/13987288/svg-image-not-visible-in-android-3-0-with-same-code/13992014#13992014

**Comment by schvenk** [comment]
Is this still an issue on 4.1/4.2? And, if you have to turn off hardware acceleration what's the impact on performance?

**Comment by pents90** [comment]
Yes, the issue seems to remain on the latest versions of Android. In fact, there are reasons to believe that there will be no fix coming, even though it seems insane that the Android team would be satisfied with having part of their API permanently broken when hardware acceleration is turned on, especially as nearly all new phones default to using hardware acceleration. Note that you can turn off hardware acceleration just for a specific view. The performance can still be good. We wrote the Androidify app using this library and it performs well even on older devices.

**Comment by schvenk** [comment]
Thanks for the clarification. I'd be looking to use it to draw the background of items in a ListView that may contain a fair number of items...any sense for whether it would be a performance issue there, vs. using PathShape or something? I love the idea of being able to draw stuff in Illustrator and just drop it in...

**Comment by pents90** [comment]
If it's just going to be a static drawing back there, should be fine. There's probably even a way to convert it into a bitmap upon layout. Then you can get full hardware acceleration.
